api/src/main/java/org/apache/cloudstack/api/response/HostResponse.java

```diff
@@ -495,6 +500,14 @@ public void setDetails(Map details) {
         detailsCopy.remove("username");
         detailsCopy.remove("password");
 
+        if(detailsCopy.containsKey(Host.HOST_UEFI_ENABLE)) {
+            this.setUefiCapabilty(Boolean.parseBoolean((String) detailsCopy.get(Host.HOST_UEFI_ENABLE)));
+            detailsCopy.remove(Host.HOST_UEFI_ENABLE);
+        } else {
+            this.setUefiCapabilty(new Boolean(false)); // in case of existing host which is not scanned for UEFI capability
```

Comment:
Can be just set to false

Reply:
Yes, this is for hosts which are already existed in CloudStack.  Auto discover Uefi capability for existing host is planned as part enhancement to this feature. Can be addressed later. Hence it's set false in initial version of implementation.